packages/bitcoin-ledger-provider/lib/BitcoinLedgerProvider.js

```diff
@@ -176,7 +176,7 @@ export default class BitcoinLedgerProvider extends LedgerProvider {
       if (inputs && outputs) {
         let change = outputs.filter(output => output.id !== 'main')
 
-        if (change) {
+        if (change && change.length > 0) {
```

Comment:
Nice catch man!

Did you get any `absurdly-high-fee` error without this?

Reply:
Sorry for late reply @harshjv. 

Yes, in fact it wouldn't actually send the transaction because the fee was too high 😂

Reply:
Adding this to separate PR: https://github.com/liquality/chainabstractionlayer/pull/186